KmsClient.js

```diff
@@ -124,24 +127,20 @@ export class KmsClient {
       capability = _getRootZcapId({keyId});
     }
 
+    const headers = await _signCapabilityInvocation({
```

Comment:
Why move this out of the try/catch?

Reply:
@dlongley - because the helper method has its own try/catch.

Reply:
@dmitrizagidulin, hmm, that's confusing to me. If `_signCapabilityInvocation` throws ... it won't go through the helper. Don't we want it to hit the `catch` and go through the helper?

Reply:
`_signCapabilityInvocation` itself uses the helper also.